pkg/cmd/roachtest/tests/ruby_pg_blocklist.go

```diff
@@ -204,6 +204,9 @@ var rubyPGIgnorelist = blocklist{
 	`PG::Connection handles server close while asynchronous connect`:                                                                                                               "flaky",
 	`PG::Connection OS thread support Connection.new shouldn't block a second thread`:                                                                                              "flaky",
 	`PG::Connection multinationalization support respect and convert character encoding of input strings should convert error string to #put_copy_end`:                             "unknown",
+	`PG::Connection in nonblocking mode rejects to send lots of COPY data`:                                                                                                         "flaky",
```

Comment:
iet's also make sure to remove these from the `blocklist` if they are present

Reply:
ah thank u for the catch